### ISSUE TITLE
Setting `content-type` lowercased makes axios prefixing 'application/x-www-form-urlencoded' to it

### ISSUE THREAD

**Issue by mattiaz9** [axios/axios#5272]
### Describe the bug

If you make an axios request with a lower case `content-type` header axios changes the header to `application/x-www-form-urlencoded, your/content-type`.

Changing the header to `Content-Type` will make the request work correctly.

Screenshot:
<img width="751" alt="Screenshot 2022-11-15 at 14 40 52" src="https://user-images.githubusercontent.com/9766700/201934131-6a76de4d-4fc6-4ad1-a6e0-e36426ac7d3e.png">

### To Reproduce

Make a request like this:
```ts
await axios.post('/api/upload', file, {
  headers: {
    'content-type': 'video/mp4',
  }
})
```

### Expected behavior

Axios should make the request with the header: `content-type: video/mp4`

### Axios Version

1.1.3

### Browser

Safari

### Browser Version

16.1

### Node.js Version

18.9

### OS

macOS Ventura

**Comment by NguyenKyThinh94** [comment]
I have same proplem. With version 1.0.0

**Comment by DigitalBrainJS** [comment]
#5067 This was fixed in `v1.2.0-alpha.1`
